python-avd/pyavd/_eos_designs/structured_config/underlay/dhcp_servers.py

```diff
@@ -0,0 +1,118 @@
+# Copyright (c) 2023-2024 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+from __future__ import annotations
+
+import re
+from functools import cached_property
+from ipaddress import AddressValueError, IPv4Address, ip_network
+
+from pyavd._errors import AristaAvdInvalidInputsError
+from pyavd._utils import get
+
+from .utils import UtilsMixin
+
+
+class DhcpServerMixin(UtilsMixin):
```

Comment:
Sorry, just to respect existing "naming convention"
```suggestion
class DhcpServersMixin(UtilsMixin):
```